pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -596,7 +596,7 @@ func getPIDString(pid *int) string {
 	return "self"
 }
 
-func (l *podNIC) loadCachedInterface() (*api.Interface, error) {
+func (l *podNIC) cachedInterface() (*api.Interface, error) {
```

Comment:
Can you rename it to `cachedDomainInterface` ?
Then I will finally manage to figure out that was cached without looking up what was originally cached.

Doing the same for the `set` one will be nice as well. It would really fit `storeCachedDomainIface`.

Reply:
OK.